src/electron.ts

```diff
@@ -116,9 +115,11 @@ app.on('activate', () => {
 
 function registerShortcuts(win: Electron.BrowserWindow): void {
```

Comment:
An idea of having an exported array in its own config file with i.e. 
```
[{keypress:'CommandOrControl+T', wckey: 'newTab', wcvalue: true}, {...}, ...]
```
Then here we include this array and iterate over using `.forEach()`.

What do u think?